packages/opc-feedback/src/opc-feedback.ts

```diff
@@ -105,14 +124,59 @@ export class OpcFeedback extends LitElement {
       crossorigin="anonymous"/>
+    
+    <!-- Bug Panel -->
+    <dialog id="bug-dialog" class="op-feedback__panel pf-u-mr-0 pf-u-display-block" .open="${this._openBugModal}">
+      <form class="bug-form" id="bugform">
+        <div class="pf-u-display-flex">
+          <ion-icon name="arrow-back-sharp" class="pf-u-mt-sm pf-u-font-size-lg" @click="${e => {
+            this._setModalState(true, false, false, true);
+            this._resetForm();
+          }}"></ion-icon>
+          <header>
+            <h3 class="pf-u-font-weight-normal pf-u-font-size-lg pf-u-text-align-center pf-u-m-xs">${this.template.errorTitle}
+            </h3>
+          </header>
+        </div>
+        ${repeat(this.template.errorList, (error) => html`
+        <div
+          class="pf-c-chip pf-m-draggable op-feedback__chip ${this._error === error.name ? 'op-feedback__chip__active' : ''}"
+          @click="${e => this._setError(error.name)}" @keydown=${e => (e.key === 'Enter') ? this._setError(error.name) : ''}
+          tabindex="0">
+          <span class="pf-u-font-size-xs">${error.name}</span>
+        </div>
+        `)}
+        <p class="op-feedback__subtitle pf-u-font-size-md pf-u-pt-md">${this.template.summary}</p>
+        <textarea id="bugsummary" rows="3" name="bugsummary"
+          @input="${(e: HTMLElementEventMap | any) => this._summary = e.target.value}"
+          placeholder=${this.template.summaryPlaceholder} class="pf-c-form-control pf-m-resize-vertical"
+          required></textarea>
+          <p class="op-feedback__subtitle pf-u-font-size-xs">
+          ${this.template.bugSubmissionNote}</p>
+        <button class="pf-c-button pf-m-block ${this._error.length === 0 ? 'pf-u-display-none-on-sm' : ''}"
+          type="button" @click="${e => {
+        this._submitFeedback();
+        this._setModalState(false, false, false, !this._openConfirmationModal);
+
+      }}">Submit</button>
+        <button id="submit-bug" class="pf-c-button pf-m-block ${this._error.length !== 0 ? 'pf-u-display-none-on-sm' : ''}"
+          type="button" disabled>Submit</button>
+      </form>
+    </dialog>
 
     <!-- Feedback Panel -->
-    <dialog id="data-dialog" class="op-feedback__panel" .open="${this._openFeedbackModal}">
-      <form class="feedback-form" id="form">
-        <header>
-          <h3 class="pf-u-font-weight-normal pf-u-font-size-lg pf-u-text-align-center pf-u-m-0">${this.template.title}
-          </h3>
-        </header>
+    <dialog id="feedback-dialog" class="op-feedback__panel pf-u-mr-0 pf-u-display-block" .open="${this._openFeedbackModal}">
+      <form class="feedback-form" id="feedbackform">
+        <div class="pf-u-display-flex">
+          <ion-icon name="arrow-back-sharp" class="pf-u-mt-sm pf-u-font-size-lg" @click="${e => {
+            this._resetForm();
+            this._setModalState(true, false,false, false);
+          }}"></ion-icon>
```

Comment:
Replace ion-icons with svgs

Reply:
What's is problem of using it.

It was there since the first version

Reply:
I removed it